src/__tests__/incidents/view/ViewIncident.test.tsx

```diff
@@ -20,61 +18,57 @@ import { RootState } from '../../../shared/store'
 const { TitleProvider } = titleUtil
 const mockStore = createMockStore<RootState, any>([thunk])
 
-describe('View Incident', () => {
-  const setup = async (permissions: Permissions[]) => {
-    jest.resetAllMocks()
-    jest.spyOn(titleUtil, 'useUpdateTitle').mockImplementation(() => jest.fn())
-    jest.spyOn(breadcrumbUtil, 'default')
-    jest.spyOn(IncidentRepository, 'find').mockResolvedValue({
-      id: '1234',
-      date: new Date().toISOString(),
-      code: 'some code',
-      reportedOn: new Date().toISOString(),
-    } as Incident)
-    const history = createMemoryHistory()
-    history.push(`/incidents/1234`)
+const setup = (permissions: any[] | undefined, id: string | undefined) => {
+  jest.resetAllMocks()
+  jest.spyOn(breadcrumbUtil, 'default')
+  jest.spyOn(IncidentRepository, 'find').mockResolvedValue({
+    id,
+    date: new Date().toISOString(),
+    code: 'some code',
+    reportedOn: new Date().toISOString(),
+  } as Incident)
+  const history = createMemoryHistory()
+  history.push(`/incidents/${id}`)
 
-    const store = mockStore({
-      user: {
-        permissions,
-      },
-    } as any)
+  const store = mockStore({
+    user: {
+      permissions,
+    },
+  } as any)
 
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
-        <ButtonBarProvider.ButtonBarProvider>
-          <Provider store={store}>
-            <Router history={history}>
-              <Route path="/incidents/:id">
-                <TitleProvider>
-                  <ViewIncident />
-                </TitleProvider>
-              </Route>
-            </Router>
-          </Provider>
-        </ButtonBarProvider.ButtonBarProvider>,
-      )
-    })
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
-  }
+  const renderResults = render(
+    <ButtonBarProvider.ButtonBarProvider>
+      <Provider store={store}>
+        <Router history={history}>
+          <Route path="/incidents/:id">
+            <TitleProvider>
+              <ViewIncident />
+            </TitleProvider>
+          </Route>
+        </Router>
+      </Provider>
+    </ButtonBarProvider.ButtonBarProvider>,
+  )
 
-  it('should set the breadcrumbs properly', async () => {
-    await setup([Permissions.ViewIncident])
+  return { ...renderResults, history }
+}
 
-    expect(breadcrumbUtil.default).toHaveBeenCalledWith([
-      { i18nKey: 'incidents.reports.view', location: '/incidents/1234' },
-    ])
-  })
+it('smoke test ViewIncidentDetails no Permissions', async () => {
```

Comment:
Sorry to be pedantic, but I think this should be 

`it('smoke tests ViewIncidentDetails when there are no permissions')` or something like that so that one case read it as _"it smoke tests ViewIncidentDetails when there are no permissions"_, I think it makes it easier to understand just by reading the description, what the test does?

Reply:
Fixed (to be merged)